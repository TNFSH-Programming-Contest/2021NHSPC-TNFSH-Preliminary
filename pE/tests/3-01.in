514 1000
3 4 25
3 4 19
5 4 8
5 4 16
5 3 7
4 6 10
6 7 30
6 7 17
5 8 9
8 7 16
7 9 19
10 5 6
10 5 24
5 11 5
11 9 18
12 13 17
9 14 22
15 13 22
9 14 29
14 16 21
9 16 5
10 16 19
10 16 14
12 15 8
16 17 5
12 13 19
13 10 25
17 18 29
12 13 23
18 19 22
13 19 6
12 13 16
19 20 16
21 12 6
20 22 11
20 22 15
20 22 5
23 24 21
24 21 12
25 23 11
23 22 24
25 23 20
26 27 21
22 26 17
25 27 15
27 28 15
27 29 27
27 29 14
25 27 21
27 29 15
28 29 28
25 27 21
30 31 12
32 33 6
25 27 28
34 33 14
27 35 20
32 33 29
36 37 28
38 39 11
29 32 27
39 40 20
35 34 21
25 27 12
41 37 19
39 40 25
42 43 17
42 43 29
39 40 19
40 44 11
45 37 11
44 30 14
42 43 11
45 41 16
39 40 18
39 40 21
36 45 19
46 47 19
39 40 29
47 48 14
42 43 7
25 33 9
38 39 12
36 49 17
37 50 22
48 25 13
50 33 23
49 46 8
31 43 8
42 38 25
42 38 29
36 46 23
42 38 29
40 43 9
42 51 18
51 43 27
42 43 5
52 42 21
33 52 28
43 53 15
36 53 25
53 54 30
36 54 21
55 36 25
55 54 30
54 56 25
57 55 6
57 56 25
58 57 25
59 60 8
58 61 25
60 62 12
58 56 9
63 56 30
58 59 23
62 56 19
58 56 22
64 65 6
66 67 6
68 56 27
61 63 20
64 65 20
65 58 10
61 56 10
69 64 8
67 70 28
71 68 7
72 70 13
69 64 27
72 66 17
73 74 20
69 56 15
72 70 16
72 70 9
69 71 21
73 75 28
75 74 9
76 77 10
72 70 25
69 56 23
78 79 18
72 73 14
70 56 9
74 76 21
72 56 22
72 56 25
72 69 26
77 80 8
74 81 13
82 79 11
81 83 20
83 80 30
82 79 12
78 79 28
78 79 17
83 80 9
80 79 30
72 79 6
72 84 10
72 56 24
72 79 24
78 79 11
56 79 22
84 82 25
72 79 23
82 79 5
78 79 27
78 72 11
78 79 7
78 79 13
85 78 27
85 79 11
85 79 19
85 79 28
79 86 19
85 86 12
85 86 6
85 86 16
85 86 10
85 86 24
87 88 26
89 90 12
88 91 5
89 90 13
92 93 27
86 92 9
91 89 10
94 87 29
86 93 6
95 96 23
94 90 26
90 85 16
96 97 26
98 94 22
99 100 29
95 101 20
102 103 5
104 105 17
105 106 6
97 101 17
104 105 11
107 108 13
100 95 8
104 105 8
106 102 25
106 107 11
93 103 28
98 93 10
106 108 26
106 103 29
106 108 19
106 103 20
109 110 7
108 99 8
106 108 12
110 98 13
106 108 8
101 103 23
104 111 26
106 103 29
106 103 14
106 101 5
111 112 16
104 109 22
112 106 14
104 103 23
113 104 25
103 114 6
114 115 12
115 116 25
117 113 16
116 118 12
118 119 24
117 119 14
117 119 23
117 119 24
119 120 14
120 121 25
122 117 12
122 121 9
122 121 22
122 121 5
121 123 8
123 124 27
122 124 20
122 124 19
125 126 20
122 125 29
122 125 18
126 124 24
127 124 16
122 124 20
122 127 10
128 122 11
124 129 30
122 124 9
128 129 26
128 129 29
128 129 8
129 130 10
130 131 27
132 133 5
133 128 5
132 131 6
134 132 27
134 131 15
135 134 18
136 137 18
137 138 12
138 139 18
140 135 10
141 136 12
141 140 26
142 143 24
144 142 16
131 139 8
143 145 30
145 141 16
139 146 16
144 146 10
147 144 6
148 146 10
148 147 27
149 148 19
146 150 10
150 151 13
152 153 27
154 152 16
154 153 21
151 154 10
153 155 25
149 153 26
149 155 21
156 149 19
149 155 21
156 149 16
157 155 11
157 155 12
158 157 30
158 157 23
156 158 7
156 155 30
156 155 25
159 156 11
160 161 15
156 155 28
159 162 26
160 161 23
155 162 22
162 160 14
159 161 27
159 161 9
159 161 13
163 164 8
164 161 26
164 161 23
159 161 8
165 166 18
163 164 14
167 168 17
169 170 6
159 161 25
163 161 17
168 169 8
171 172 23
172 161 17
159 163 19
159 171 6
165 166 19
166 167 24
161 165 29
173 159 7
174 173 24
174 170 15
174 170 18
170 175 15
174 175 22
176 177 15
177 178 27
175 176 8
177 178 5
178 179 26
174 179 22
179 180 30
180 181 6
181 182 15
183 184 18
184 174 6
184 181 10
181 182 12
182 185 30
186 183 16
186 185 5
186 185 9
186 185 14
186 185 15
187 186 24
187 188 13
187 188 15
187 185 19
188 189 26
187 189 5
190 185 5
187 190 20
187 189 14
185 189 15
187 189 6
187 189 19
191 187 26
191 189 21
192 193 27
193 191 27
191 189 24
194 192 27
191 189 13
189 195 24
194 192 12
192 195 15
194 192 17
196 197 8
195 196 15
197 198 11
194 198 10
199 194 11
200 201 12
198 202 17
200 203 22
201 203 25
202 204 14
199 202 14
199 202 11
202 204 24
204 200 8
199 202 26
205 199 21
206 207 21
205 199 28
208 209 17
210 211 18
209 210 16
208 212 23
205 199 19
208 213 23
214 202 24
215 216 18
199 214 27
217 218 9
217 218 20
206 207 29
218 205 11
215 216 15
218 203 16
208 213 6
219 207 25
206 219 5
211 213 27
218 203 30
218 203 9
216 213 27
208 215 12
212 213 16
217 218 20
208 213 19
217 203 5
220 221 11
217 203 25
213 222 19
217 203 20
221 223 5
224 203 25
225 226 25
203 220 11
213 222 25
227 225 20
227 226 7
223 228 22
206 207 10
217 224 12
203 221 25
203 221 23
229 203 9
230 208 13
207 231 15
231 229 27
232 227 15
233 232 14
206 231 18
234 226 30
235 236 15
222 237 24
238 237 6
238 230 6
217 206 11
239 240 9
228 239 10
217 203 9
234 226 19
235 241 14
234 226 10
226 238 8
236 241 24
242 217 20
240 243 19
234 226 18
242 203 17
244 245 14
246 247 11
248 244 18
234 226 22
248 249 29
249 245 17
248 245 15
226 237 10
237 235 12
241 246 19
234 250 13
250 247 23
234 233 25
234 226 20
241 246 12
234 251 25
252 248 28
247 251 18
245 253 27
254 255 13
242 243 26
252 253 22
255 256 6
256 242 19
253 243 14
257 258 17
259 257 24
252 234 28
258 243 10
234 251 21
252 260 22
261 262 22
263 261 13
264 243 10
252 259 18
260 264 12
265 266 8
251 243 7
267 268 25
269 270 17
270 265 6
268 254 27
252 267 19
269 270 6
271 243 20
270 266 12
252 267 22
266 272 20
273 272 25
270 266 30
266 273 19
269 270 24
252 243 24
252 271 11
269 270 17
266 273 17
274 262 20
252 243 9
243 269 23
275 252 25
275 262 16
272 263 9
275 274 15
275 262 24
275 262 23
275 262 16
275 262 28
275 262 21
275 262 26
275 262 7
275 262 26
275 262 8
262 276 10
275 276 6
275 276 23
275 276 11
276 277 21
275 277 18
275 277 29
275 278 11
275 278 11
277 278 27
275 279 21
279 278 14
280 281 24
282 283 9
284 285 30
286 280 8
285 278 24
284 285 21
286 280 6
285 275 18
283 287 8
288 289 13
289 290 9
285 291 12
281 288 25
292 293 11
278 294 8
291 294 9
290 292 25
286 293 26
286 293 19
282 287 5
295 296 27
295 296 10
295 296 16
297 298 16
294 297 22
299 300 15
286 293 27
282 287 10
284 298 5
286 301 14
301 302 5
293 302 20
303 304 8
304 305 22
306 307 15
308 303 15
309 307 13
309 306 15
308 310 28
309 296 24
299 300 17
284 298 27
311 295 22
309 307 30
284 312 27
302 313 7
298 312 13
299 300 5
309 307 29
310 314 12
309 307 12
282 287 10
315 282 10
314 304 24
284 312 6
309 284 23
309 312 6
299 300 19
309 311 11
286 313 18
316 317 14
309 316 23
315 304 27
307 296 28
287 304 8
299 300 26
318 296 18
309 318 14
319 286 27
320 299 22
309 317 15
312 296 16
296 315 15
313 321 28
322 305 16
323 313 29
309 317 27
323 324 10
323 313 5
322 305 9
296 315 26
324 319 20
315 308 10
313 321 15
309 317 6
325 322 18
326 323 26
296 305 7
323 321 6
309 317 26
327 325 15
317 300 5
326 323 28
317 320 26
320 300 15
326 328 11
328 329 17
309 317 22
321 330 16
330 329 17
326 329 15
300 327 20
331 332 14
333 334 18
326 329 27
305 333 25
326 329 13
331 335 22
336 337 30
326 329 20
338 339 29
331 332 22
337 305 26
335 332 23
340 341 18
339 309 9
329 341 6
305 342 21
343 336 19
344 345 19
338 343 22
346 347 23
342 334 13
346 344 8
326 340 28
348 338 11
348 338 19
347 344 25
344 349 12
332 350 17
349 345 25
350 348 12
331 350 14
345 351 7
341 334 23
331 334 19
352 331 28
353 354 24
326 352 9
334 355 9
352 331 16
355 346 9
354 356 13
346 351 11
351 353 12
326 334 8
326 356 5
356 357 21
326 357 27
357 358 22
358 359 17
326 359 11
360 326 30
359 361 13
361 362 22
363 362 20
364 363 26
364 362 26
364 362 22
360 364 11
360 365 24
366 360 14
365 362 23
367 368 24
366 362 21
367 368 10
366 362 26
367 368 29
369 362 16
370 367 23
366 362 6
370 368 25
371 372 21
369 362 30
366 369 15
373 374 27
372 373 21
368 375 12
375 371 21
366 362 8
376 377 25
373 374 23
378 379 25
370 375 5
380 381 11
382 376 18
383 375 7
374 366 16
383 370 16
362 384 15
385 382 21
383 375 26
383 375 16
370 375 12
379 383 27
377 386 21
384 385 26
383 375 7
386 387 21
378 388 18
378 383 19
388 389 21
389 383 20
390 380 20
373 374 24
378 388 9
381 391 7
392 393 5
392 393 20
383 394 22
394 373 19
386 387 17
392 390 23
386 387 29
393 391 30
390 381 17
378 386 9
390 381 7
378 386 13
386 387 14
395 378 28
392 391 7
395 378 9
386 387 6
387 396 27
392 391 6
386 396 20
397 398 25
397 398 17
397 398 10
397 398 19
391 395 29
399 400 28
398 392 6
401 402 17
403 392 10
400 402 10
404 405 17
400 401 20
398 403 10
398 386 28
399 400 24
402 397 22
406 396 11
406 396 7
405 399 8
404 405 12
406 396 23
397 398 5
398 406 20
396 407 7
408 409 26
407 408 25
404 408 27
404 408 5
404 408 5
410 411 27
411 412 7
412 404 8
413 410 7
413 409 7
413 409 12
414 415 30
415 413 19
409 416 30
414 417 10
416 418 23
417 418 26
414 419 9
418 420 12
419 420 18
419 420 19
421 414 15
420 422 26
423 421 30
424 422 6
425 423 27
424 423 8
424 425 29
424 422 28
424 422 23
424 422 15
426 427 20
424 422 8
424 422 20
424 422 25
427 428 15
426 427 26
429 422 8
430 426 14
429 424 21
426 427 19
422 430 25
429 430 27
428 431 28
432 429 6
432 431 24
433 432 21
434 433 7
431 435 30
434 435 20
434 435 16
436 434 30
435 437 14
438 436 28
438 437 19
437 439 13
439 440 16
441 438 10
440 442 10
442 443 14
443 444 13
441 444 27
443 444 27
445 444 8
441 445 12
446 447 24
447 448 12
448 441 25
441 444 29
444 449 26
449 450 26
450 451 21
446 449 28
451 452 18
452 453 19
453 454 20
446 453 25
454 455 15
455 456 14
456 457 28
458 457 26
446 458 16
446 457 17
446 457 9
459 460 23
461 460 11
446 457 14
462 461 18
462 459 7
463 462 6
457 460 26
463 462 12
446 457 27
446 457 29
463 460 29
463 460 15
463 446 11
463 460 21
463 460 12
463 460 13
464 463 16
463 460 22
465 464 16
466 467 9
468 460 8
467 469 11
468 460 11
470 471 30
466 469 25
472 473 28
473 474 15
475 470 16
468 472 11
474 465 5
475 466 10
476 2 18
471 469 14
476 2 30
477 468 25
460 478 5
479 480 18
477 481 12
481 479 28
469 482 7
475 469 25
475 469 13
480 478 29
476 477 17
483 484 8
484 469 29
483 475 5
476 2 23
469 482 14
478 2 8
477 478 11
485 486 15
476 2 13
487 482 13
488 482 12
482 489 25
489 476 10
476 2 18
490 491 5
487 488 26
487 483 13
485 486 14
476 2 7
492 490 9
476 2 6
482 476 30
485 486 22
488 482 14
487 493 14
494 485 21
495 496 8
486 495 17
492 491 28
491 496 10
487 482 6
497 492 13
498 499 25
482 2 21
500 498 6
493 501 13
500 498 28
487 493 30
500 498 27
498 496 18
502 499 16
498 499 19
499 496 15
498 497 12
498 494 13
487 2 12
500 498 15
501 2 25
498 502 22
503 2 26
487 2 29
504 505 13
504 506 16
505 506 11
507 500 6
498 496 5
506 487 14
507 508 22
509 510 19
508 503 20
496 2 16
503 511 8
507 2 25
511 504 10
512 513 22
510 507 11
1 514 28
513 509 26
507 2 5
514 513 20
1 512 7
1 513 30
