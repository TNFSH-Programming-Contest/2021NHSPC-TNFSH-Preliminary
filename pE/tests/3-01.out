70.69961174
